CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# viash_project_template 0.3.0
+
+* Update to Viash 0.8.2
+
+    * `demo_pipeline` is now built from `src/workflows/demo_pipeline`
+    * State management overhaul
+    * Use of helper functions (dependencies) from [`vsh_utils`](https://viash-hub.com/data-intuitive/vsh_utils)
```

Comment:
Might I suggest using a more specific name than `vsh_utils`? ATM, its mostly viash-*nextflow* related utilities. But from the name it could be anything: wrappers around the viash CLI (like [viash-tools](https://github.com/viash-io/viash_tools)), plugins for viash,... ) Since its available on viash-hub I think we should be more careful with the names in order to be as specific as possible and to avoid namespace clashes later on. WDYT?

Reply:
TBD

Reply:
`vsh_pipeline_operators`

Reply:
See [vsh-pipeline-operators](https://viash-hub.com/data-intuitive/vsh-pipeline-operators)